src/Components/Shop/ShopList.jsx

```diff
@@ -0,0 +1,30 @@
+    { id: 1, imagesUrl: ["https://www.tate.org.uk/art/images/work/L/L01/L01682_10.jpg"], name: "Product name", price: 100, description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum hendrerit odio a erat lobortis auctor. Curabitur sodales pharetra placerat. Aenean auctor luctus tempus. Cras laoreet et magna in dignissim. Nam et tincidunt augue. Vivamus quis malesuada velit. In hac habitasse platea dictumst." },
+    { id: 1, imagesUrl: ["https://www.tate.org.uk/art/images/work/L/L01/L01682_10.jpg"], name: "Product name", price: 100, description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum hendrerit odio a erat lobortis auctor. Curabitur sodales pharetra placerat. Aenean auctor luctus tempus. Cras laoreet et magna in dignissim. Nam et tincidunt augue. Vivamus quis malesuada velit. In hac habitasse platea dictumst." },
+    { id: 1, imagesUrl: ["https://www.tate.org.uk/art/images/work/L/L01/L01682_10.jpg"], name: "Product name", price: 100, description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum hendrerit odio a erat lobortis auctor. Curabitur sodales pharetra placerat. Aenean auctor luctus tempus. Cras laoreet et magna in dignissim. Nam et tincidunt augue. Vivamus quis malesuada velit. In hac habitasse platea dictumst." },
+    { id: 1, imagesUrl: ["https://www.tate.org.uk/art/images/work/L/L01/L01682_10.jpg"], name: "Product name", price: 100, description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum hendrerit odio a erat lobortis auctor. Curabitur sodales pharetra placerat. Aenean auctor luctus tempus. Cras laoreet et magna in dignissim. Nam et tincidunt augue. Vivamus quis malesuada velit. In hac habitasse platea dictumst." },
+    { id: 1, imagesUrl: ["https://www.tate.org.uk/art/images/work/L/L01/L01682_10.jpg"], name: "Product name", price: 100, description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum hendrerit odio a erat lobortis auctor. Curabitur sodales pharetra placerat. Aenean auctor luctus tempus. Cras laoreet et magna in dignissim. Nam et tincidunt augue. Vivamus quis malesuada velit. In hac habitasse platea dictumst." },
+    { id: 1, imagesUrl: ["https://www.tate.org.uk/art/images/work/L/L01/L01682_10.jpg"], name: "Product name", price: 100, description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum hendrerit odio a erat lobortis auctor. Curabitur sodales pharetra placerat. Aenean auctor luctus tempus. Cras laoreet et magna in dignissim. Nam et tincidunt augue. Vivamus quis malesuada velit. In hac habitasse platea dictumst." }]
```

Comment:
I guess this is the way you are mocking the values for testing?
The way frontend-first is to be used is to create all the needed services that will comunicate with the future api - but with the api calls replaced with a file source that contains json.
That way we will already have a fully functional frontend with services to communicate, the only thing needed to be replaced will be the actual calls to the api.